client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/site-php-version.tsx

```diff
@@ -3,7 +3,7 @@ import { useTranslate } from 'i18n-calypso';
 export default function SitePhpVersion( {
 	phpVersion,
 }: {
-	phpVersion: number | null;
+	phpVersion: string | null;
```

Comment:
The `php_version_num` field is a number returned by the endpoint

Reply:
Reverted back to number in fb88090.